systemd-swap

```diff
@@ -424,7 +442,13 @@ case "$1" in
                                 DEV=$(GET_WHAT_FROM_SWAP_UNIT $UNIT_PATH)
                                 UNIT_NAME=$(basename $UNIT_PATH)
                                 INFO "Zram: swapoff $DEV"
-                                swapoff "$DEV"
+                                swapoff "$DEV" &
```

Comment:
Not im not, im placed wait right after the loop, so it will wait for all actions to be done. It has to be async because of 2 reasons:
1. single thread(device) of deflate is MUCH slower than lz4
2. amount of data could be stored in ram MUCH higher (and there is no tradeoff)
Thats why we have to swapoff in parallel or it could take hour (im not kidding, it taking a lot for my 20GB zram swap)

Reply:
You're right, im trying to wait inside loop, thank you! Misstype